snuba/settings_base.py

```diff
@@ -85,7 +85,4 @@
 
 PROJECT_STACKTRACE_BLACKLIST = set()
 
-# Number of queries each subscription consumer can run concurrently.
-SUBSCRIPTIONS_MAX_CONCURRENT_QUERIES = 10
```

Comment:
I'm not sure that we can realistically give a sensible default limit for this right now.

When this gets going, it would probably be a good idea to keep a number of threads reserved for subscription query processing to maintain quality of service and avoid backlogging on these queries. The napkin math way to calculate this value would be to figure out the total number of database threads dedicated to subscription processing for the dataset, and divide that number by the number of consumer group members to get the number of query workers per consumer.

I haven't really come up with a model for capacity planning how many threads should be allocated for subscriptions, but the total number of threads available on the consistent nodes and the number of datasets per database cluster are factors, as are the number and frequency of subscriptions per consumer and the duration of those subscription queries.

I suspect that this value will need to be scaled differently per dataset, and vary wildly with the environment (SaaS will use more threads and consumers than single tenant which will use more threads and consumers than on-premise.)